### PR TITLE
Feature/au 001

### PR DESCRIPTION
**Pull request by siosond**

### CODE REVIEW

**Comment by lewisken02** [comment]
nuks

**Comment by angeloPereyra** [comment]
We should use `[]` instead of `array()` when initializing arrays to shorten our code

File: app/Http/Controllers/Rest/AuthRestController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class AuthRestController extends Controller
+{
+    //
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function login(Request $request)
```
Comment (siosond) [inline app/Http/Controllers/Rest/AuthRestController.php]: naming convention

File: app/Http/Controllers/Rest/AuthRestController.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace App\Http\Controllers\Rest;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class AuthRestController extends Controller
+{
+    //
+    public function __construct()
```
Comment (siosond) [inline app/Http/Controllers/Rest/AuthRestController.php]: comments

File: resources/js/utils/validations.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * detects if given variable is a string and it has
+ * intended number of characters, from iMin to iMax
+ * (inclusive)
+ *
+ * @param mVar
+ * @param iMin
+ * @param iMax
+ * @returns {boolean}
+ */
+window.isStringValid = function (mVar, iMin=0, iMax=0) {
```
Comment (siosond) [inline resources/js/utils/validations.js]: does this check whitespaces and alt 255

File: resources/js/chat/chat.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+new Vue({
+    el: '#public_chat_div',
+    data: {
+        message: ''
+    },
+    methods: {
+        sendChat: function (e) {
+            alert('You want to send this message: "' + this.message + '"?');
```
Comment (siosond) [inline resources/js/chat/chat.js]: wat dis is

File: resources/views/welcome.blade.php [hunk 5]
```diff
@@ -1,98 +1,39 @@
-<!doctype html>
-<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<html>
     <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (siosond) [inline resources/views/welcome.blade.php]: isama

**Pull request merged**